torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -84,13 +84,16 @@ std::pair<TypePtr, bool> MergeInferredType(
 void MergeInferredTypeAndSetMap(
     Value* dest_v,
     TypePtr existing_type,
-    TypePtr inferred_type) {
+    TypePtr inferred_type,
+    bool set_map) {
```

Comment:
Can you add some comments about this boolean and when to set it?

Reply:
Or maybe just rename the boolean to set_constant_value_map, and add the comment in torch/csrc/jit/passes/onnx.cpp  Line 297 why it is set to false (which is to avoid redundancy).